tools/imageregistry/daemonset.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: setup-script
+  namespace: kube-system
```

Comment:
Is it possible to put these objects in a namespace other than `kube-system`? I wonder if we could isolate this in something like `rukpak-e2e`?

That way, we could blow away the namespace and get the garbage collection of everything in it.

Reply:
It's probably possible.  I'll try.

Reply:
I moved the registry into `rukpak-e2e` namespace